### DISCUSSION TITLE
Why evaluates on cpu

### DISCUSSION THREAD

**Discussion by 9p15p** [open-mmlab/mmengine#1419] [Q&A]
https://github.com/open-mmlab/mmengine/blob/eb4fa73b56535be07f8fe113957da2526906ca37/mmengine/evaluator/metric.py#L132

it runs slowly on cpu. If I want to use GPU to speed up the evaluation. What should I do?

**Comment by YiyaoYang1** [comment]
Sorry, these calculations cannot be accelerated with GPU. In this stage, the sample category labels and classification predictions are extracted from the results to calculate the classification accuracy (acc). This calculation can only be performed on CPU.

**Reply by 9p15p** [reply]
Thanks for your reply.

I am building my project(not classification) based on mmengine
I think it's a good idea to give users an option to complete time-consuming measurements on GPU.
